rewards/mainnet_allocations.csv

```diff
@@ -0,0 +1,1420 @@
+﻿Account Name,Time,Keyset Name,Quantity,Chain ID

```

Comment:
What's the difference between this file and the allocations YAML files?

Reply:
this is the data used to generate the yamls